dts/arm64/qemu/qemu-virt-a53.dtsi

```diff
@@ -37,6 +37,18 @@
 			compatible = "arm,cortex-a53";
 			reg = <1>;
 		};
+
+		cpu@2 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <2>;
+		};
+
+		cpu@3 {
+			device_type = "cpu";
+			compatible = "arm,cortex-a53";
+			reg = <3>;
+		};
```

Comment:
`qemu-virt-a53.dtsi` is included by both `qemu_cortex_a53.dts` and `qemu_cortex_a53_smp.dts`. With this, now we have 4 CPU devicetree nodes in `qemu_cortex_a53` that only has one CPU.

cefba05a71af8d2b10c2eace1f082d9e595c59c0 is incorrect and should be reverted -- this only "worked" because it is not currently used to infer anything.

The additional CPU nodes should be added in `qemu_cortex_a53_smp.dts`.